tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -79,6 +79,22 @@ services:
   {% set container_postfix = loop.index %}
     - "awx_{{ container_postfix }}"
   {% endfor %}
+{% endif %}
+{% if enable_keycloak|bool %}
+  keycloak:
+    image: quay.io/keycloak/keycloak:15.0.2
```

Comment:
I'd like to add `container_name`, because I don't like the default name it gets of `sources_keycloak_1`.

I think maybe `tools_keycloak` would be best, but ultimately up to you.

Reply:
I'd like to keep the `_1` to match the naming convention of `tools_postgres_1` and `tools_redis_1` but I like changing it from `sources_` to `tools_`.

Reply:
we have a `tools_receptor_hop` container, that's where I was coming from here. But the `_1` is fine too.